app/assets/stylesheets/components/avatars.scss

```diff
@@ -0,0 +1,54 @@
+@import '../vars/variables';
+
+@mixin avatar-logo($name, $radius, $background) {
+  .crayons-#{$name} {
+    display: inline-block;
+    border-radius: $radius;
+    position: relative;
+    background-color: $background;
+    width: $su-6;
+    height: $su-6;
+
+    &__image {
+      border-radius: $radius;
+      width: 100%;
+      height: 100%;
+      vertical-align: bottom; // todo: do we need it?
+    }
+
+    &::after {
```

Comment:
This adds a subtle inner shadow for when we have white logo/avatar on white background. Seems hacky but felt so much better than just applying outside box-shadow. And you can really add inner box-shadow on the image so...